.github/workflows/integration-tests.yml

```diff
@@ -89,7 +89,7 @@ jobs:
 
       - name: Spawn Exasol environment
         run: |
-          ./start-test-env spawn-test-environment --environment-name test --database-port-forward 8563
+          ./start-test-env spawn-test-environment --environment-name test --database-port-forward 8563 --deactivate-database-setup
```

Comment:
did you intend to commit this?

Reply:
Yes. This speeds up the build and we don't need it.
Tests only need a DB, no JDBC driver installed.

See https://github.com/exasol/integration-test-docker-environment/blob/master/README.md#options: 
> Deactivates the setup of the spawned database, this means no data get populated and no JDBC drivers get uploaded.